apps/web/pages/api/user/avatar.ts

```diff
@@ -11,6 +12,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const username = req.query.username as string;
   const teamname = req.query.teamname as string;
   let identity;
+  let linksToThisRoute = false;
```

Comment:
```suggestion
  let isAvatarSourceWebAppUrl = false;
```

Reply:
This might not be a better variable name considering it can be WEBSITE_URL